conda_vendor/conda_vendor.py

```diff
@@ -1,218 +1,499 @@
+""" The main idea is to pass an isolated environment (environment.yaml)  to
+ conda-lock and get a list of packages, locations, and metadata.  We
+ download all the packages specified into a solution into the directory
+ structure of a Conda channel.  We grab the repodata for each package from
+ the original source and write a condensed repodata.json only having our
+ vendored packages.
+ """
 import click
-import yaml
-import sys
-import struct
-import requests
 import hashlib
 import json
-from conda_vendor.version import __version__
-from conda_vendor.conda_lock_wrapper import CondaLockWrapper
-from conda_lock.src_parser import LockSpecification
-from conda_lock.conda_solver import DryRunInstall, VersionedDependency, FetchAction
+import requests
```

Comment:
Are these import deletions done for pylint?

Reply:
no.  there was just a lot of detritus, so it's cleaned up.